cli/show.h

```diff
@@ -13,7 +13,7 @@
 namespace show
 {
 
-void physicalPort()
+inline void physicalPort()
```

Comment:
Fixing: `Function 'physicalPort' defined in a header file; function definitions in header files can lead to ODR violations`